frontend/src/components/common/SideBar/style.ts

```diff
@@ -0,0 +1,95 @@
+import { Classes } from "@blueprintjs/core";
+import { LIGHT_GRAY, PT_TEXT_COLOR } from "src/components/common/theme";
+import styled from "styled-components";
+
+interface Props {
+  sideBarWidth: number;
+}
+
+interface PositionerProps {
+  isExpanded: boolean;
+}
+
+export const SideBar = styled.div<Props>`
+  box-sizing: border-box;
+  box-shadow: inset -1px 0px 0px rgba(16, 22, 26, 0.15);
+  width: ${(props: Props) => `${props.sideBarWidth}px`};
```

Comment:
Since we already did `styled.div<Props>`, we can omit type at call site, I believe?
```suggestion
  width: ${(props) => `${props.sideBarWidth}px`};
```

here and throughout